### DISCUSSION TITLE
YouTube Downloader

### DISCUSSION THREAD

**Discussion by rudymohammadbali** [TomSchimansky/CustomTkinter#2161] [Showcase]

**Comment by dipeshSam** [comment]
> <h1 align="center"><a href="https://github.com/iamironman0/youtube_downloader" target="_blank">YouTube Downloader</h1>
> 
> <p align="center">“Experience a modern, user-friendly YouTube downloader app. Built with Pytube and CustomTkinter, this GUI application offers a seamless experience for downloading YouTube videos and playlists.”</p>
> 
> ###
> 
> <h2 align="left">Demo</h2>
> 
> ###
> 
> <p align="center">
>   <img src="https://github.com/iamironman0/youtube_downloader/assets/63475761/94ae3d76-fa63-420f-b784-33df6ffeb269">
> </p>

Nice!
